report.Rmd

```diff
@@ -188,39 +349,42 @@ ggplot(standard_dictator_game_data, aes(x = treatment_group, y = Mean, fill = tr
 ```
 
 ```{r Figure 2 Mean_SD, echo=FALSE}
-# Explain what the Kruskal-Wallis test is applied on
-cat("Mean and SD for the distributions of 'kept' across different treatment groups within the Standard Dictator Game.\n")
-# Print the adjusted data
-print(standard_dictator_game_data)
+# Mean and SD for the distributions of 'kept' across different treatment groups within the Standard Dictator Game.
+kable(standard_dictator_game_data) %>% kable_styling()
 ```
 
 ```{r Figure 2 KW, echo=FALSE}
-# Explain what the Kruskal-Wallis test is applied on
-cat("Performing Kruskal-Wallis test to compare the distributions of 'kept' across different treatment groups within the Standard Dictator Game.\n")
+# Performing Kruskal-Wallis test to compare the distributions of 'kept' across different treatment groups within the Standard Dictator Game.
 
 # Conduct Kruskal-Wallis test
 kruskal_test <- kruskal.test(kept ~ treatment_group, data = df %>% filter(game_type == "Standard Dictator Game"))
 
 # Store the p-value for use in the plot annotation
 p_value <- round(kruskal_test$p.value, 4)
 
-# Create an annotation text with the Kruskal-Wallis result
-annotation_text <- paste("Kruskal-Wallis, p =", p_value, ". Note. Confidence intervals on means are at 95%.")
-print(annotation_text)
+paste("Kruskal-Wallis, p =", p_value)
```

Comment:
Here we can add "Kruskal-Wallis for Standard Dictator Game, p =" to have more visibility on results